### DISCUSSION TITLE
Something which I can not use helix current version to replace nvim/vim

### DISCUSSION THREAD

**Discussion by dlintw** [helix-editor/helix#9014] [General]
After 10 days study and use of helix.  I found there are something which is better for me to use nvim/vim.

1.  repeat in vim "." is often useful than which in helix.  I wish there is a switch to let the repeat simulate vim.
2. completion in vim "ctrl-p", "ctrl-n" is useful and fast, In helix's Insert mode, it is impossible use it for normal text.  Sometimes, what I just need is repeat the long word which I type in previous lines.
3. Still don't know how to enable spelling check in comment or normal text file.
4. Without [VOoM](https://vim-voom.github.io/) plugin support, It is difficult to jump proper section to continue my writing.
5. Without vim-diff and vim-diffdir function.

However, I like helix builtin support for TS, LSP and launch speed, that's simplified terrible monthly experience for configure nvim/vim.  The typing speed is slow down about 10% after switching from nvim to helix because its default key mapping is defined be longer than vim's.  And it is impossible to define back to what vim did.

**Comment by dead10ck** [comment]
Hi, thanks for trying out helix. There are open issues for most if not all of these issues already. Generally, big laundry lists of feature requests like this aren't very actionable or helpful. In the future, I recommend searching through existing issues and discussions before posting.
